package.json

```diff
@@ -37,6 +37,7 @@
     "graphql-relay": "^0.6.0",
     "graphql-tools": "^4.0.7",
     "graphql-upload": "10.0.0",
+    "http2": "^3.3.7",
```

Comment:
remove this

Reply:
this was added by accident when installing locally... removed

Reply:
Can you revert the package-lock.json

Reply:
Done